.github/workflows/build.yml

```
@@ -648,27 +467,43 @@ jobs:
           set -eux
 
           # Replace version
-          sed -i "s/__version__ = \"latest\"/__version__ = \"${{ env.VOICEVOX_ENGINE_VERSION }}\"/" voicevox_engine/__init__.py
-
-          pyinstaller \
-            --noconfirm \
-            --collect-data pyopenjtalk \
-            --add-data "licenses.json;." \
-            --add-data "presets.yaml;." \
-            --add-data "default.csv;." \
-            --add-data "engine_manifest.json;." \
-            --add-data "speaker_info;speaker_info" \
-            --add-data "engine_manifest_assets;engine_manifest_assets" \
-            --add-binary "download/onnxruntime/lib/onnxruntime.dll;." \
-            --add-binary "download/core/core.dll;." \
-            run.py
+          if [[ ${{ matrix.os }} == macos-* ]]; then
+            gsed -i "s/__version__ = \"latest\"/__version__ = \"${{ env.VOICEVOX_ENGINE_VERSION }}\"/" voicevox_engine/__init__.py
+            pyinstaller \
+              --noconfirm \
+              --collect-data pyopenjtalk \
+              --add-data "licenses.json:." \
+              --add-data "presets.yaml:." \
+              --add-data "default.csv:." \
+              --add-data "engine_manifest.json:." \
+              --add-data "speaker_info:speaker_info" \
+              --add-data "engine_manifest_assets:engine_manifest_assets" \
+              --add-binary "download/onnxruntime/lib/libonnxruntime.dylib:." \
+              --add-binary "download/core/libcore.dylib:." \
+              run.py
+          else
+            sed -i "s/__version__ = \"latest\"/__version__ = \"${{ env.VOICEVOX_ENGINE_VERSION }}\"/" voicevox_engine/__init__.py
+            pyinstaller \
```

これ気づけないので、一旦こんな感じでコメントで案内するのどうでしょう。。こんな感じで。
```suggestion
            # windowsの場合はセミコロンが区切り文字
            pyinstaller \
```